hcl/hcl.go

```diff
@@ -361,12 +364,25 @@ func (p *TerramateParser) MinimalParse() error {
 	return errs.AsError()
 }
 
+// ParsedBodies returns a map of filename to the parsed hclsyntax.Body.
+func (p *TerramateParser) ParsedBodies() map[string]*hclsyntax.Body {
```

Comment:
While I was starting to add the new map of parsed blocks I realized we already have this method, should be enough for the cloning feature :100: